cypress/support/skipLiveDescribe.js

```diff
@@ -0,0 +1,7 @@
+const skipLiveDescribe = (name, func) => {
```

Comment:
Maybe add a comment to this file explaining that this is necessary because Persian isn't live yet. When Persian is live, we can remove this hack.

Reply:
done :)